pkg/plugins/golang/v2/scaffolds/internal/templates/controllers/controller_suitetest.go

```diff
@@ -126,7 +126,7 @@ package controllers
 import (
 	"path/filepath"
 	"testing"
-	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/v2"
```

Comment:
why this change should be done for the go/v2 plugin? 
Also, is the `"github.com/onsi/ginkgo/v2"` really compatible with the dependence versions used by go/v2 and when we scaffold the removed apis v1beta1 for CRDs and webhooks?

Reply:
> when we scaffold the removed apis v1beta1 for CRDs and webhooks?

I'm not understand why this change is associated with `v1beta1 for CRDs and webhooks`? 😂

Reply:
reverted~